AzureSQLConnectivityChecker.ps1

```diff
@@ -69,6 +88,16 @@ if ($null -ne $parameters) {
     }
 }
 
+# Setting default parameters if not provided
+
+if ($null -eq $AuthenticationType -or '' -eq $AuthenticationType) {
+    $AuthenticationType = 'SQL Server Authentication'
+}
```

Comment:
I see that these variables are set on the beginning of the file to some default values. Maybe these are unnecessary checks?

Reply:
Or you can move them up

Reply:
There are two options: 1. to set parameters inside the script directly, or 2. to call a script and provide the paramters. So if someone uses the script directly and doesn't set the parameters, the would be null. So keeping this.